pub-critic/src/app/page.module.scss

```diff
@@ -0,0 +1,96 @@
+.Container {
+    padding: 70px 5vw;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    background-color: #151515;
+    .Page {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        width: 100%;
+        .Image {
+            width: 100vw;
+            height: 100vh;
+            position: relative;
+            > img {
+                width: 100%;
+                height: 100%;
+                object-fit: cover;
```

Comment:
I think you can set this directly in Next Image

Reply:
:white_check_mark: pretty sure this works without the styles